src/plugins/data/public/ui/query_string_input/language_switcher.tsx

```diff
@@ -104,8 +104,8 @@ export function QueryLanguageSwitcher(props: Props) {
             <FormattedMessage
               id="data.query.queryBar.syntaxOptionsDescription"
               defaultMessage="The {docsLink} (DQL) offers a simplified query
-              syntax and support for scripted fields. DQL also provides autocomplete if you have
-              a Basic license or above. If you turn off DQL, OpenSearch Dashboards uses Lucene."
+              syntax and support for scripted fields. If you turn off DQL,
```

Comment:
Was autocomplete removed?

Reply:
I was asking specifically about the autocomplete feature. Does DQL not provide autocomplete after the fork?

Reply:
Yeah it's part of the x-pack stuff, seems like it's been a feature request to implement a substitute for a while https://github.com/opendistro-for-elasticsearch/sample-code/issues/131

Reply:
Okay, good to know.